src/App.tsx

```diff
@@ -48,46 +62,86 @@ export function getReorderedGoods(
 // };
 
 export const App: React.FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
+
+  const reverseGoods = () => {
+    setIsReversed(!isReversed);
+  };
+
+  const sortAlphabetically = () => {
+    setSortType(SortType.ALPHABET);
+  };
+
+  const sortByLength = () => {
+    setSortType(SortType.LENGTH);
+  };
+
+  const reset = () => {
+    setSortType(SortType.NONE);
+    setIsReversed(false);
+  };
+
+  const visibleGoods = getReorderedGoods(goodsFromServer, {
+    sortType,
+    isReversed,
+  });
+
   return (
     <div className="section content">
       <div className="buttons">
         <button
           type="button"
-          className="button is-info is-light"
+          className={
+            sortType === SortType.ALPHABET
+              ? 'button is-info'
+              : 'button is-info is-light'
+          }
```

Comment:
classnames library

Reply:
Now I see. I was working late at night, not the best idea 😃